Patches/Vanilla Animals Expanded/Australia_Animals.xml

```diff
@@ -10,6 +10,13 @@
       <operations>
         <!-- === Boombat === -->
         <li Class="PatchOperationAdd">
+			<xpath>Defs/ThingDef[defName="Boomalope"]/tradeTags</xpath>
```

Comment:
This this supposed to be `AEXP_Boombat` here?

Reply:
yes... I knew i missed something